packages/venia-ui/lib/queries/createAccount.graphql

```diff
@@ -11,12 +11,11 @@ mutation createAccount(
             lastname: $lastname
             password: $password
         }
-    ) {
+    ) @connection(key: "createCustomer") {
```

Comment:
Shouldn't we just use `no-cache` here? I don't see a reason to cache the response at all.

Reply:
As it stands now, yes. Selecting only the `id` field isn't going to update the `customer` cache entry.

I opted for the `@connection` directive instead in case we ever select more than just the `id` field in the future. Our cache entry for `customer` wouldn't get updated if we had a `no-cache` policy set.

Reply:
Talked offline: we're going to be intentional about using `no-fetch`

Reply:
Resolved in https://github.com/magento/pwa-studio/pull/2240/commits/c52080c9e4bad80d1f8f7aa42f8c964f1ff7d26b.